.github/workflows/e2e-ci.yml

```diff
@@ -40,8 +40,18 @@ jobs:
         run: git diff --exit-code
       - name: Check E2E wallet balance
         run: NODE_OPTIONS='--unhandled-rejections=strict' yarn ts-node ./e2e/scripts/check-e2e-wallet-balance.ts
+      - name: Java version
+        run: java -version
+      - name: Profile and then java version
+        run: |
+          source ~/.profile
+          java -version
+      - name: Java version on bash
+        run: java -version
+        shell: bash
       - name: Create Detox Build
         run: |
+          source ~/.profile
```

Comment:
This seems shouldn't be necessary. I assume it's for debugging the issue we're facing with our runners.

Reply:
yep, for now leaving this on till we figure out the issue with getting java 11 as the default version to unblock other work